modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/definitions/portlet/action/test/GetObjectDefinitionInfoMVCResourceCommandTest.java

```diff
@@ -0,0 +1,148 @@
+import com.liferay.portal.kernel.test.portlet.MockLiferayResourceRequest;
+import com.liferay.portal.kernel.test.portlet.MockLiferayResourceResponse;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.JavaConstants;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+import com.liferay.portal.workflow.constants.WorkflowDefinitionConstants;
+import com.liferay.portal.workflow.kaleo.service.KaleoDefinitionLocalService;
+
+import java.io.ByteArrayOutputStream;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Nathaly Gomes
+ */
+@RunWith(Arquillian.class)
+public class GetObjectDefinitionInfoMVCResourceCommandTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Test
+	public void testGetObjectDefinitionInfo() throws Exception {
+		ObjectDefinition objectDefinition =
+			ObjectDefinitionTestUtil.addCustomObjectDefinition();
+
+		String workflowDefinitionName = RandomTestUtil.randomString();
+
+		_kaleoDefinitionLocalService.addKaleoDefinition(
+			RandomTestUtil.randomString(), workflowDefinitionName,
+			workflowDefinitionName, null, null,
+			WorkflowDefinitionConstants.SCOPE_ALL, 1,
+			ServiceContextTestUtil.getServiceContext());
+
+		_workflowDefinitionLinkService.addWorkflowDefinitionLink(
+			TestPropsValues.getUserId(), TestPropsValues.getCompanyId(),
+			TestPropsValues.getGroupId(), objectDefinition.getClassName(), 0, 0,
+			workflowDefinitionName, 1);
+
+		JSONObject jsonObject = _getObjectDefinitionInfoJSONObject(
+			objectDefinition.getObjectDefinitionId());
+
+		Assert.assertEquals(
+			JSONUtil.put(
+				"tableName", objectDefinition.getDBTableName()
+			).put(
+				"workflowDefinitionLabel", workflowDefinitionName
+			).toString(),
+			jsonObject.toString());
+	}
+
+	private JSONObject _getObjectDefinitionInfoJSONObject(
+			long objectDefinitionId)
+		throws Exception {
+
+		MockLiferayResourceRequest mockLiferayResourceRequest =
+			new MockLiferayResourceRequest();
+
+		ThemeDisplay themeDisplay = new ThemeDisplay();
+
+		themeDisplay.setCompany(
+			_companyLocalService.fetchCompany(TestPropsValues.getCompanyId()));
+		themeDisplay.setSiteGroupId(TestPropsValues.getGroupId());
+
+		themeDisplay.setUser(TestPropsValues.getUser());
+
+		mockLiferayResourceRequest.setAttribute(
```

Comment:
Can you sort to have this order below?

```
mockLiferayResourceRequest.addParameter("objectDefinitionId"..			
mockLiferayResourceRequest.setAttribute(JavaConstants.JAVAX_PORTLET_CONFIG..
mockLiferayResourceRequest.setAttribute(WebKeys.THEME_DISPLAY..
```
The themeDisplay should be created just before invoking `setAttribute(WebKeys.THEME_DISPLAY` or in-lining calling a private method `_getThemeDisplay()`

Reply:
Hey @vngoliveira, I think you forgot to change the last part of my comment about the `themeDisplay`.